bdk_chain/src/keychain/keychain_txout_index.rs

```diff
@@ -3,6 +3,7 @@ use crate::{
     miniscript::{Descriptor, DescriptorPublicKey},
     ForEachTxout, SpkTxOutIndex,
 };
+use alloc::borrow::ToOwned;
```

Comment:
NIT: We don't need to use `ToOwned` as we are just going from `&T` to `T`.

Reply:
Went ahead with a clone..